maestro-cli/src/main/java/maestro/cli/command/TestCommand.kt

```diff
@@ -659,6 +707,38 @@ class TestCommand : Callable<Int> {
         )
     }
 
+    private fun parseLogLevels(): Set<LogLevel>? {
```

Comment:
This looks arduous. I've seen other implementations set a threshold, e.g. "INFO" means "INFO and above". Would that work here?